web/custom-plugins/forecast-of-response/src/Pages/OperationCreate.tsx

```diff
@@ -137,11 +138,25 @@ const onClickCreate = (
   const getIds = []
   // Prepare the uncontained entities for the Operation
   operationLocation.type = Blueprints.LOCATION
-  getIds.push(getEntityId(operationLocation, token, isNewEntity.location))
+  getIds.push(
+    getEntityId(
+      operationLocation,
+      token,
+      isNewEntity.location,
+      'FoR-Data/Locations'
```

Comment:
maybe add new variables to const.ts with values for "'FoR-Data/Locations'" and 'FoR-Data/Configs'?

Reply:
Fixed